lib/mixlib/install/product.rb

```diff
@@ -198,18 +185,6 @@ def lookup(key, version = :latest)
     config_file "/etc/opscode/chef-server.rb"
   end
 
-  product "chef-server-ha-provisioning" do
```

Comment:
@patrick-wright @schisamo quite the opposite, this is an EA product.  I don't know if a build has gone through since the cutover from packagecloud.   @charlesjohnson is the best person to talk to.